extension/externalsubscriber/server.go

```diff
@@ -27,12 +27,12 @@ var (
 // TODO: switch to eventually using chain.Stateless block
```

Comment:
```suggestion
// TODO: switch to eventually using chain.Stateful block
```

Reply:
This is as it should be. We should switch to using `chain.StatelessBlock` + `chain.Result` ideally in a single type like `ExecutedBlock`, so we don't need to define a type to forward the data here.